src/kudu/tserver/simple_tablet_manager.cc

```diff
@@ -160,6 +337,16 @@ Status TSTabletManager::Init() {
   VLOG(2) << "RaftConfig before starting: " << SecureDebugString(consensus_->CommittedConfig());
 
   scoped_refptr<Log> log;
+
+  OpId init;
+  init.set_term(0);
+  init.set_index(0);
+  scoped_refptr<MetricEntity> metric_entity;
+  metric_entity = METRIC_ENTITY_server.Instantiate(metric_registry_, kSysCatalogTabletId);
```

Comment:
Why can't we re-use the metric entity of the server?

Reply:
I think this was confusion between  METRIC_ENTITY_tablet and METRIC_ENTITY_server.